src/flows/components/header/index.tsx

```diff
@@ -60,7 +60,7 @@ const FlowHeader: FC = () => {
     if (isFlagEnabled('shareNotebook')) {
       getNotebooksShare({query: {orgID: '', notebookID: flow.id}})
         .then(res => {
-          if (res.data) {
+          if (!!res?.data?.length) {
```

Comment:
this is annoying. `length` technically isn't a property of `Shares` even though `Shares = Share[]`. Thanks TS